src/pages/admin/home.tsx

```diff
@@ -12,5 +12,5 @@ const AdminHomePage = () => {
     </>
   );
 };
-
+AdminHomePage.requireAuth = true;
```

Comment:
Is there a chance we make this true once the URL starts with "/admin"?

Reply:
I fail to see how this can be achieved